code/modules/reagents/Chemistry-Recipes.dm

```diff
@@ -3556,5 +3556,68 @@
 	new /mob/living/simple_animal/parrot(location)
 	qdel(holder.my_atom)
 
+/datum/chemical_reaction/ectoplasm
```

Comment:
I am not a fan of being able to create ectoplasm with just:
1) a thing that can be created without leaving chemistry
2) a thing from Dan's vending machine
3) a thing from the Groans vending machine
I wouldn't mind it as much if it didn't create an unlimited amount of hostile skeleton mobs. I feel like something's got to give somewhere.

As an addendum for the less autistic reader: the reagent would still be obtainable by grinding solid ectoplasm obtained by killing spooky/cult mobs. It would be very rare if that was the only source, so some kind of middle ground would still allow ectoplasm products to be powerful/interesting while not being produced 5 minutes into any round.

Reply:
I balanced it against carp synth. I produced it normally while testing and it took a bit even with my practice making carp synth quickly. I do get the concern but let me go over my thought process:

- Aminomicin takes a while to make. It's one of the more complex chems. The universal enzyme is a big limiter because of the nutriment and producing less than 3u. 

- The amounts of bone marrow and frost oil you get from vending machines won't be enough for even tiny amounts. It's very time consuming to gamble on getting the frost oil soda especially. It requires 3u bone marrow to make which was intended to make grinding up bones the best option. Bones are (arguably) spookier than Dan's.

- Keep in mind ectoplasm also only produces 1u from all 3 reagents. So you get a 3:1 ratio when making aminomicin then a 5:1 ratio making ectoplasm. You get very little of the stuff for your efforts. It's possible to produce a lot more with exotic condiment crates but that also adds a lot of time to the production. 

- 10u degen calcium is another limiter. It can take a while to heat that much and while milk is easy to get, it's not gotten from the dispenser.

- Finally, carp synth is a better option for mob loosing:
Skeletons do 2-5 damage a hit and have 50 health. They do nothing special aside from that and cannot be picked up.
Carp do 15 damage a hit and have 25 health. They reproduce into 4-6 baby carp very quickly and can be picked up.
Baby carp do 8 damage a hit and have 15 health. A monkey in a pit makes them grow up immediately as they attack corpses. They can be picked up and around 8 can fit in a cardboard box.
Most importantly skeletons do not have some kind of skeleton pheromone. Carp pheromone allows you to survive your own loose. Skeletons will bone you every time.

It's going to take way more than 5 minutes to make even 1 skeleton. Even when you do make it the number isn't unlimited. You'll likely get a dozen or so after half an hour. That's about 1 egun and 1 cap laser worth of skeleton.

Reply:
You can already make synthetic carp without even leaving medbay. I don't see how making weak skeletons like this is too powerful considering the other shit you can pull with chemistry.

Reply:
I'll let this go because you make a convincing case, but not because I'm convinced. I still think spawning a bunch of hostile mobs is dumb, regardless of whether another dumb reaction already does it (also known as whataboutism).

Reply:
Thank you. If it causes problems it's easy to tinker with later. Like I said I have a lot of plans for the chem over the course of the month so I might want to make it harder to make to give me more freedom with those anyway. I think it's already fairly difficult though.